Feature/new getting started (#2228)
mercadopago/devsite-docs: guides/additional-content/getting-started/getting-started-v2.en.md

If you already have a Mercado Libre account, access [Your integrations](https://www.mercadopago.com/developers/panel/app) with the same username and password.

# Your integrations

With your account already active, you have access to [Your integrations](https://www.mercadopago.com/developers/panel/app), which allows you to manage code development integrations, your applications, perform tests, and configure notifications that provide you with information about transactions.

---
bullet_section_with_media:
 - type: reverse
 - title: Get to know our payment solutions
 - message: Mercado Pago has different payment solutions for each type of business. Find them below:
 - image: /getting-started/integration.png
---

## Process online payments

> Solutions for those who sell on the website, through WhatsApp, or social media.

Reviewer: ```suggestion
Solutions for those who sell on websites, through WhatsApp, or social media.
```

----[mlb]----
| Solution | Description | Integration Complexity |
|:---|:---|:---|
| Checkout Pro | Process payments in Mercado Pago in a simple and secure way with a ready-to-use design. We have developed this optimized checkout to increase your conversion. | 🔵🔵⚪⚪ |

> | Checkout Transparent | A versatile solution so that you build your payment processor on your website. You control the entire experience, from basic to advanced settings. | 🔵🔵🔵🔵 |

Reviewer: ```suggestion
| Checkout Transparente | A versatile solution so that you build your payment processor on your website. You control the entire experience, from basic to advanced settings. | 🔵🔵🔵🔵 |
```

> | Payment Link | Create a payment link in just a few steps and use it to collect payments via chat and social networks. No technical knowledge is required. | 🔵⚪⚪⚪ |

Reviewer: ```suggestion
| Payment Link | Create a payment link in just a few steps and use it to collect payments via chat and social media. No technical knowledge is required. | 🔵⚪⚪⚪ |
```

------------

## Offer in-person payments

Solutions for those who sell from home, on the street, in the car, or in the store.

----[mla]----
| Solution | Description | Integration Complexity |
|:---|:---|:---|
| QR Code | Choose an integration model to generate QR codes and provide a complete in-person payment experience. | 🔵🔵⚪⚪ |
| Mercado Pago Point | Integrate the Mercado Pago Point and connect the card reader to any iOS or Android system to accept payments at the counter. | 🔵🔵🔵⚪ |
| Mercado Pago Delivery | A on-demand food delivery solution in which the customer accesses a list of registered restaurants, places an order, and pays directly through the Mercado Pago app. | 🔵🔵⚪⚪ |
------------

----[mlb, mlm, mpe, mco, mlu, mlc]----
| Solution | Description | Integration Complexity |
|:---|:---|:---|
| QR Code | Choose an integration model to generate QR codes and provide a complete in-person payment experience. | 🔵🔵⚪⚪ |
| Mercado Pago Point | Integrate the Mercado Pago Point and connect the card reader to any iOS or Android system to accept payments at the counter. | 🔵🔵🔵⚪ |
------------

## Process payments on e-commerce platforms

> Install Checkout Pro or ----[mla, mlm, mpe, mco, mlu, mlc]----Checkout API ------------ ----[mlb]----Checkout Transparent ------------ in your store and **start offering payments with Mercado Pago** on your e-commerce platform. Integrating solutions by connecting to a platform is simple and fast. The entire process is done in the Mercado Pago business panel and on the platform itself.

Reviewer: ```suggestion
Install Checkout Pro or ----[mla, mlm, mpe, mco, mlu, mlc]----Checkout API ------------ ----[mlb]----Checkout Transparente ------------ in your store and **start offering payments with Mercado Pago** on your e-commerce platform. Integrating solutions by connecting to a platform is simple and fast. The entire process is done in the Mercado Pago business panel and on the platform itself.
```

Find our plugin on platforms like **Shopify**, **WooCommerce**, **Tiendanube**, **VTEX**, **Wix**, **Adobe Commerce (Magento)**, and many more.

## Use our SDKs

With Mercado Pago's SDK libraries, you can reduce client-side and server-side integration time.

By using our **backend SDKs**, you get easy access to the server-side functionality of our solutions. This allows you to create and query the status of various transactions, integrate payments through cards and other methods, as well as make refunds or chargebacks.

With our **frontend SDKs**, you can make calls for card tokens and use many other integration features.

<br>

## Availability of solutions in each country

----[mlb]----
| Solution | 🇦🇷 <br> Argentina | 🇧🇷 <br> Brazil | 🇨🇱 <br> Chile | 🇨🇴 <br> Colombia | 🇲🇽 <br> Mexico | 🇵🇪 <br> Peru | 🇺🇾 <br>Uruguay |
| :--- | :--- | :--- | :--- | :--- | :--- | :--- | :--- |
| Checkout Pro | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ |

> | Transparent Checkout | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ |

Reviewer: ```suggestion
| Checkout Transparente | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ | ✔ |
```
